views/default/tab_client_reissue_cert.pdt

```diff
@@ -105,6 +105,24 @@ if($use_central_api != true )
                         </p>
                     <?php
                     }
+					else if($auth_file_name == 'fileauth.txt'){
+						?>
+						<p>
+							For File Based Authentication, Please create a folder structure "/.well-known/pki-validation/" under root directory and create file using following details or download the Authentication file by clicking the "Download Auth File" button and place it in the "/.well-known/pki-validation/" of the HTTP server, like so: http://www.example.com/.well-known/pki-validation/<?php $this->Html->_($auth_file_name); ?> It may take some time for file to be automatically validated and cert to be issued by Certificate Authority.

```

Comment:
There is a lot of capitalization in this sentence that might not be proper, particularly '... , Please ...'.  Also there is a missing period after one of your uris 'Like so: ... <?...?>.'